components/style/stylist.rs

```diff
@@ -76,9 +76,85 @@ struct DocumentCascadeData {
     precomputed_pseudo_element_decls: PerPseudoElementMap<Vec<ApplicableDeclarationBlock>>,
 }
 
-impl DocumentCascadeData {
```

Comment:
Thanks, I re-run my upgrade script after the rebase (rather than before) and everything is fine now.